BigCommerce/Integration/04_ManageProductData.md

```diff
@@ -0,0 +1,99 @@
+# Manage product and offer data
+
+Before you start exchanging data with *BigCommerce*, you must check your product and offer data.
+You must check the images assigned to an offer in the *PIM* or the *Omni-Channel* module. In addition, you can define the color-swatch variant option.
+
+
+## Check product images
+
+Verify that the desired product data image is in the correct position and that the correct text field is maintained.   
+
+It is recommended for a good presentation of the offers in your store that the desired images are displayed for each variant. *BigCommerce* supports the import of one thumbnail per variant only. The first image in the corresponding attribute is marked as thumbnail image for the offer. Additionally, the alt-text of that image will be added as description.   
+The following procedure describes how to check the images in the *PIM* module. If you only have the *Omni-Channel* module in use, you can check the image accordingly via *Offers > Select offer > Edit offer*.
+
+
+#### Prerequisites
+
+You have assigned more than one image per product variant.
+
+#### Procedure
+
+*PIM > Products > Tab LIST*
+
+![Product images](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/PIMCheckImages.png "[Product images]")
+
+1. Click the product variant you want to check and click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit) button.   
+  The *Edit product* view is displayed. The *Attributes* tab is preselected.
+
+    ![Edit product](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditProduct.png "[Edit product]")
+
+2. Click the *Files and Images* entry of the *Basic Data* attribute group or the customer-specific attribute group in which you have defined the images.   
+   The *Files and images* section is displayed. 
+
+   ![Files and images](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/PIMImageAndFiles.png "[Files and images]")
+
+3. Verify that the image at the first position is suitable for your *BigCommerce* store. If not, drag and drop the desired image to the first position.
+
+4. Verify that the *Alt text* field is filled for the image on first position. If you must change the existing entry, you have the following options: 
+    - You can change the entry. This might have impact on other connections you have in use.
```

Comment:
Ich finde diese Infos etwas widerspruchlich... (you *must* change... you *can* change... you *want to keep unchanged*... Vielleicht anders formulieren?

4. Verify that the *Alt text*... If necessary, you can change the entry, but bear in mind that this might have an impact in other connections you have in use. Alternatively, if you want to keep the product data in this scope unchanged, ...

Reply:
hm